site/content/docs/5.0/components/navs.md

```diff
@@ -187,14 +187,14 @@ Force your `.nav`'s contents to extend the full available width one of two modif
 </ul>
 {{< /example >}}
 
-When using a `<nav>`-based navigation, be sure to include `.nav-item` on the anchors.
+When using a `<nav>`-based navigation.
```

Comment:
This seems off now

Reply:
Unable to understand @XhmikosR 😊
Can you please define?

Reply:
The sentence seems weird with that part removed :) Maybe it should be reworded?

Reply:
Or you can even use a `<nav>`-based navigation.
@XhmikosR?